rustls-platform-verifier/src/android.rs

```diff
@@ -68,33 +68,35 @@ fn global() -> &'static Global {
         .expect("Expect rustls-platform-verifier to be initialized")
 }
 
-/// Initializes and stores the required context for the Android platform.
-///
-/// This method will setup and store an environment locally. This is useful if
-/// nothing else in your application needs access the Android runtime.
+/// Initialize given a typical Android NDK `JNIEnv` and `jobject` context.
```

Comment:
nit: `jobject` -> `JObject` in this docstring to match the casing of the type being ref'd.